ocw/lib/gce.py

```diff
@@ -11,10 +11,13 @@ class GCE(Provider):
     def __new__(cls, vault_namespace):
         if vault_namespace not in GCE.__instances:
             GCE.__instances[vault_namespace] = self = object.__new__(cls)
-            self.__compute_client = None
-            self.__project = None
+            self.init()
         return GCE.__instances[vault_namespace]
 
+    def init(self):
```

Comment:
```suggestion
    def __init__(self):
```
and than you don't need to explicitly call it in __new__

Reply:
done